tests/rbd_mirror/rbd_mirror_reconfigure_secondary_cluster.py

```diff
@@ -0,0 +1,114 @@
+"""Test case covered -
+    CEPH-9477- Secondary cluster permanent failure,
+    Recreate the secondary cluster and Re-establish mirror with newly created cluster as secondary.
+
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+    (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+    conf and keyring files
+
+    Test case flows:
+    1) After site-b failure, bring up new cluster for secondary
+    2) Create a pool with same name as secondary pool
+    3) Perform the mirroring bootstrap for cluster peers
+    4) copy and import bootstrap token to peer cluster
+    5) verify peer cluster got added successfully after failover
+    6) verify all the images from secondary mirrored to primary
+"""
+import datetime
+import time
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """
+    Verification of rbd mirroring failover scenario on secondary site,
+    This module configures the mirroring to newly created cluster.
+
+    Args:
+        kw: test data
+        Example::
+            config:
+                poolname: pool_failback
+    Returns:
+        int: The return value - 0 for success, 1 for failure
+    """
+    try:
+        log.info("Running test CEPH-9477")
```

Comment:
here also add test headlines so it's easy in logs.

Reply:
done.